src/main/scala/io/github/pauljamescleary/petstore/infrastructure/endpoint/OrderEndpoints.scala

```diff
@@ -7,56 +7,62 @@ import io.circe.syntax._
 import org.http4s._
 import org.http4s.circe._
 import org.http4s.dsl.Http4sDsl
-import scala.language.higherKinds
 
+import scala.language.higherKinds
 import domain.OrderNotFoundError
+import io.github.pauljamescleary.petstore.domain.authentication.Auth
 import domain.orders.{Order, OrderService}
+import io.github.pauljamescleary.petstore.domain.users.User
+import tsec.authentication.{AugmentedJWT, SecuredRequestHandler, asAuthed}
+import tsec.jwt.algorithms.JWTMacAlgo
 
-class OrderEndpoints[F[_]: Effect] extends Http4sDsl[F] {
-
-  /* Need Instant Json Encoding */
-  import io.circe.java8.time._
+class OrderEndpoints[F[_]: Effect, Auth: JWTMacAlgo] extends Http4sDsl[F] {
 
-  /* Needed for service composition via |+| */
-  import cats.implicits._
+  import cats.implicits._ // |+|
+  import alias._
 
   /* Needed to decode entities */
-  implicit val orderDecoder = jsonOf[F, Order]
+  implicit val orderDecoder: EntityDecoder[F, Order] = jsonOf[F, Order]
```

Comment:
i like adding the type annotation here, and it means you can drop the ascriptions from `jsonOf`.